data_types/softlayer_virtual_guest.go

```diff
@@ -35,7 +35,8 @@ type SoftLayer_Virtual_Guest struct {
 	PrimaryBackendIpAddress string `json:"primaryBackendIpAddress,omitempty"`
 	PrimaryIpAddress        string `json:"primaryIpAddress,omitempty"`
 
-	Location *SoftLayer_Location `json:"location"`
+	Location   *SoftLayer_Location `json:"location"`
+	Datacenter *SoftLayer_Location `json:"datacenter"`
```

Comment:
Did I leave this out on purpose previously? Just wondering why this was not in from the start? I assume this is in the current SL documentation for the APIs.